Src/Plugins/Tests.Security/Validators/SEC101_027.MailChimpApiKeyValidatorTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Helpers;
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Newtonsoft.Json;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Validators
+{
+    /// <summary>
+    /// Testing SEC101/027.MailChimpApiKeyValidator
+    /// </summary>
+    public class MailChimpApiKeyValidatorTests
```

Comment:
Please consider adding [ExcludeFromCodeCoverage] attribute to all the test classes to take their code out of the Code Coverage computation.

Reply:
done